src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -5,32 +5,50 @@
 import nextstep.subway.line.domain.LineRepository;
 import nextstep.subway.line.dto.LineRequest;
 import nextstep.subway.line.dto.LineResponse;
+import nextstep.subway.section.domain.Section;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.domain.StationRepository;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
 import java.util.stream.Collectors;
 
 import static nextstep.subway.exception.ApiExceptionMessge.NOT_FOUND_LINE;
+import static nextstep.subway.exception.ApiExceptionMessge.NOT_FOUND_STATION;
 
 @Service
 @Transactional
 public class LineService {
-    private final LineRepository lineRepository;
+    private final LineRepository lines;
```

Comment:
```suggestion
    private final LineRepository lineRepository;
```
변수명을 lines라고 쓰면 repository라고 인식하기 어렵습니다

Reply:
2주차 제이슨님의 강의를 보고 한번 적용해봤습니다. Repository를 컬렉션으로도 볼 수 있다고 하셔서 lines로 해봤는데요.. 평소에 Repository를 붙여서 쓰는 편이라 말씀하신대로 다시 수정해봤습니다 ( 791bc22 )